### DISCUSSION TITLE
cleanupHttpCache: invalid cache data on Bitbucket Server

### DISCUSSION THREAD

**Discussion by amezin** [renovatebot/renovate#28432] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Bitbucket Server, Docker image ghcr.io/renovatebot/renovate:37.296.0-full

### Please tell us more about your question or problem

Since a recent update I'm getting "cleanupHttpCache: invalid cache data":

- once for every repository if RENOVATE_REPOSITORY_CACHE=enabled

- twice if RENOVATE_REPOSITORY_CACHE=reset

- no warnings if RENOVATE_REPOSITORY_CACHE=disabled

Removing all directories doesn't seem to help.

### Logs (if relevant)

<details><summary>Logs</summary>

```
DEBUG: validateReconfigureBranch() (repository=TEST/repo)

DEBUG: No reconfigure branch found (repository=TEST/repo)

 WARN: cleanupHttpCache: invalid cache data (repository=TEST/repo)

DEBUG: Removing any stale branches (repository=TEST/repo)

DEBUG: config.repoIsOnboarded=true (repository=TEST/repo)
```

</details>

**Comment by mjnowen** [comment]
Related. Borked PR (2 days ago): https://github.com/renovatebot/renovate/discussions/28439

**Comment by rarkins** [comment]
Converted to issue: #28440
